crates/conformance-test-utils/src/madara.rs

```diff
@@ -0,0 +1,163 @@
+use reth_primitives::Bytes;
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_storage_var_address;
+
+use crate::types::{ContractAddress, Felt, StorageKey, StorageValue};
+
+pub fn genesis_load_bytecode(
+    bytecode: Bytes,
+    address: FieldElement,
+) -> Vec<((ContractAddress, StorageKey), StorageValue)> {
+    bytecode
+        .chunks(16)
+        .enumerate()
+        .map(|(i, x)| {
+            let mut storage_value = [0u8; 32];
+            storage_value[16..16 + x.len()].copy_from_slice(x);
+            let storage_value = FieldElement::from_bytes_be(&storage_value).unwrap().into(); //safe unwrap since x.len() == 16
```

Comment:
can't we initialize a `storage_value` with only 16 bytes and avoid the `unwrap`?